plugins/google-analytics/src/lib/google-analytics.js

```diff
@@ -0,0 +1,8 @@
+const ReactGA = require('react-ga');
+
+const googleAnalytics = module.exports;
+
+googleAnalytics.init = (window) => {
+    ReactGA.initialize('UA-000000-01');
```

Comment:
I wonder how can we make this id configurable without exposing server-side secrets into the client code 🤔

Reply:
What if the id can be passed as an extra arg when emitting the event on page render. @fmvilas what do you think?